examples/matching-engine/src/lib.rs

```diff
@@ -256,18 +258,18 @@ Owner 2 should now get back their tokens, and have 145 FUN2 left. On
 query {
     accounts {
         entry(
-            key: "User:90d81e6e76ac75497a10a40e689de7b912db61a91b3ae28ed4d908e52e44ef7f"
+            key: "User:$OWNER_2"
         ) {
             value
         }
     }
 }
 ```
 
-[chain1_matching_engine]: http://localhost:8080/chains/e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65/applications/e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65060000000000000000000000e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65080000000000000000000000
-[chain2_matching_engine]: http://localhost:8080/chains/e54bdb17d41d5dbe16418f96b70e44546ccd63e6f3733ae3c192043548998ff3/applications/e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65060000000000000000000000e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65080000000000000000000000
-[chain2_fun1]: http://localhost:8080/chains/e54bdb17d41d5dbe16418f96b70e44546ccd63e6f3733ae3c192043548998ff3/applications/e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65000000000000000000000000e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65020000000000000000000000
-[chain2_fun2]: http://localhost:8080/chains/e54bdb17d41d5dbe16418f96b70e44546ccd63e6f3733ae3c192043548998ff3/applications/e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65030000000000000000000000e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65050000000000000000000000
+[chain1_matching_engine]: http://localhost:8080/chains/$CHAIN_1/applications/$MATCHING_ENGINE
+[chain2_matching_engine]: http://localhost:8080/chains/$CHAIN_2/applications/$MATCHING_ENGINE
+[chain2_fun1]: http://localhost:8080/chains/$CHAIN_2/applications/$FUN1_APP_ID
+[chain2_fun2]: http://localhost:8080/chains/$CHAIN_2/applications/$FUN2_APP_ID
```

Comment:
Those links also won't work anymore in the README files. We probably just shouldn't make them links anymore, and just show the URL (with the to-be-substituted variables).